lib/tasks/db_maintenance.rake

```diff
@@ -835,6 +872,61 @@ namespace :cartodb do
       log(end_message, task_name, database_host)
     end
 
+    # Executes a ruby code proc/block on all existing users who are dedicated or organization owners, outputting some info
+    # @param task_name string
+    # @param block Proc
+    # @example:
+    # execute_on_dedicated_users_or_organization_owners_with_index(:populate_new_fields.to_s, Proc.new { |user, i| ... })
+    def execute_on_dedicated_users_or_organization_owners_with_index(task_name, block, num_threads=1, sleep_time=0.1, database_host=nil)
+      puts 'actest'
+      if database_host.nil?
+        puts "NO database host"
+        count = ::User.with_sql("select users.* from users, organizations where users.organization_id=organizations.id and (users.account_type = '[DEDICATED]' OR users.id = organizations.owner_id)").count
```

Comment:
Some dedicated users may not have an organization. This should be a left join on organizations to pick up dedicated users with null `organization_id`.